src/services/compare-mmdl/serverless.yml

```diff
@@ -148,6 +152,7 @@ functions:
       stage: ${sls:stage}
       sesLogGroupName: ${param:sesLogGroupName}
       errorsTopicArn: ${param:errorsTopicArn}
+      seatoolLink: ${param:seatoolLink}
```

Comment:
is this supposed to be `seatoolSubdomain`?

Reply:
omg yes, thank you!